src/components/EditFeatureDrawer/EditFeatureForm/index.tsx

```diff
@@ -37,13 +50,40 @@ export type EditFeatureFormProps = FormProps & {
   form: FormInstance;
 };
 
+export interface ShogunFile extends BaseEntity {
+  active?: boolean;
+  fileName?: string;
+  fileType?: string;
+  fileUuid?: string;
+}
```

Comment:
yes, that works. Couldn't find `File` before, because there is also https://developer.mozilla.org/en-US/docs/Web/API/File

See https://github.com/terrestris/shogun-gis-client/pull/1127/commits/552c7d0271bf894675fc6a55dd6f9e22fb717242